prime-angular-frontend/src/app/modules/paper-enrolment/pages/regulatory-page/regulatory-form.model.ts

```diff
@@ -1,3 +1,5 @@
-import { HttpEnrollee } from '@shared/models/enrolment.model';
+import { CollegeCertification } from '@enrolment/shared/models/college-certification.model';
 
-export interface RegulatoryForm extends Pick<HttpEnrollee, 'certifications'> { }
+export interface RegulatoryForm {
```

Comment:
Was there a reason to remove the `extend Pick<...>` from the model?

Reply:
By doing this you create a cross module dependency instead of just using the shared parent model, and associated keys